Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "2.7"
+
+[dev-packages]
+
+
+
+[packages]
+
+"flake8" = "*"
+"flake8-isort" = "*"
+funcsigs = "*"
```

Comment:
This shouldn't be needed anymore as it's a pytest dependency and pipenv should pick it up. Can you try removing it? This list should be the direct dependencies only, and Pipfile.lock will be the complete list.